datadriven/src/test_data.rs

```diff
@@ -0,0 +1,106 @@
+use std::fmt;
+
+/// CmdArg contains information about an argument on the directive line. An
+/// argument is specified in one of the following forms:
+/// - arg
+///     - key: arg
```

Comment:
This is confusing. I thought `key: arg` is a part of the form. I think following should self-explained:
```
- key
- key=
- key=a # single_value
- key=a,b,c # multiple value
- key=(a,b,c) # same as above.
```

Reply:
Can I delete argument type `- a,b,c`, I don't think we need this.
Only the above will remain.

Reply:
LGTM